content/image.js

```diff
@@ -0,0 +1,28 @@
+const fs = require("fs");
+const path = require("path");
+
+const { ROOTS } = require("./constants");
+const { memoize, slugToFolder } = require("./utils");
+
+function urlToFilePath(url) {
+  const [, locale, , ...slugParts] = url.split("/");
+  return path.join(locale.toLowerCase(), slugToFolder(slugParts.join("/")));
+}
+
+const find = memoize((relpath) => {
```

Comment:
nit: either `relPath` or imo preferably `relativePath`. My mantra is no abbridging you wouldn't do in normal speech (which legitimizes `id`).